openfpga_flow/tasks/fpga_verilog/dsp/multi_mode_mult_16x16/config/task.conf

```diff
@@ -16,15 +16,17 @@ timeout_each_job = 20*60
 fpga_flow=yosys_vpr
 
 [OpenFPGA_SHELL]
-openfpga_shell_template=${PATH:OPENFPGA_PATH}/openfpga_flow/openfpga_shell_scripts/fix_heterogeneous_device_example_script.openfpga
+openfpga_shell_template=${PATH:OPENFPGA_PATH}/openfpga_flow/openfpga_shell_scripts/bitstream_setting_example_script.openfpga
```

Comment:
I suggest to not touch the fpga-verilog test case. 
Since this is a feature for fpga-bitstream, I suggest to add a new test case under https://github.com/lnis-uofu/OpenFPGA/tree/master/openfpga_flow/tasks/fpga_bitstream

The test case name can be ``overload_dsp_mode_bits`` which reflects your exact purpose.

Let me know what you think.